delegation-toolkit/get-started/quickstart.md

```diff
@@ -45,7 +45,8 @@ const bundlerClient = createBundlerClient({
 
 [Create a MetaMask smart account](../how-to/create-smart-account/index.md) to send the first user operation.
 
-This example configures a [Hybrid](../how-to/create-smart-account/configure-accounts-signers.md#configure-a-hybrid-smart-account) smart account:
+This example configures a [Hybrid](../concepts/smart-accounts.md#hybrid-smart-account) smart account,
+which is a flexible smart account implementation that supports both an externally owned account (EOA) owner and any number of P256 (passkey) signers:
```

Comment:
Same comment for P256

Reply:
I think we can keep p256, since we use that in concepts. I got confused with the code.